backend/PrintAsm.ml

```diff
@@ -0,0 +1,103 @@
+(* *********************************************************************)
+(*                                                                     *)
+(*              The Compcert verified compiler                         *)
+(*                                                                     *)
+(*          Xavier Leroy, INRIA Paris-Rocquencourt                     *)
+(*          Bernhard Schommer, AbsInt Angewandte Informatik GmbH       *)
+(*                                                                     *)
+(*  Copyright Institut National de Recherche en Informatique et en     *)
+(*  Automatique.  All rights reserved.  This file is distributed       *)
+(*  under the terms of the INRIA Non-Commercial License Agreement.     *)
+(*                                                                     *)
+(* *********************************************************************)
+
+open AST
+open Asm
+open Camlcoq
+open Datatypes
+open PrintAsmaux
+open Printf
+open Sections
+open TargetPrinter
+
+module Target = (val (sel_target ()):TARGET) 
+
+module Printer(Target:TARGET) =
+  struct
+
+    let print_location oc loc =
+      if loc <> Cutil.no_loc then Target.print_file_line oc (fst loc) (snd loc)
+          
+    let print_function oc name fn =
+      Hashtbl.clear current_function_labels;
+      Target.reset_constants ();
+      let (text, lit, jmptbl) = Target.get_section_names name in
+      Target.section oc text;
+      let alignment =
+        match !Clflags.option_falignfunctions with Some n -> n | None -> 4 in
```

Comment:
The default alignment for the code of a function should ideally be part of the Target structure.  In the current CompCert, it is 16 for IA32 and 4 for PowerPC and ARM.  Those values are those used by GCC, and I don't know if they make a real difference performance-wise.  But in principle it is for each Target to declare its default code alignment.